neon_tts_plugin_mozilla_remote/__init__.py

```diff
@@ -0,0 +1,84 @@
+# NEON AI (TM) SOFTWARE, Software Development Kit & Application Development System
+# All trademark and other rights reserved by their respective owners
+# Copyright 2008-2021 Neongecko.com Inc.
+#
+# Redistribution and use in source and binary forms, with or without modification, are permitted provided that the
+# following conditions are met:
+# 1. Redistributions of source code must retain the above copyright notice, this list of conditions
+#    and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions
+#    and the following disclaimer in the documentation and/or other materials provided with the distribution.
+# 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote
+#    products derived from this software without specific prior written permission.
+
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+# INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+from urllib.parse import urlencode
+from urllib.request import urlopen
+from neon_utils.configuration_utils import get_neon_tts_config
+from neon_utils.logger import LOG
+from neon_utils.parse_utils import format_speak_tags
+try:
+    from neon_audio.tts import TTS, TTSValidator
+except ImportError:
+    from mycroft.tts import TTS, TTSValidator
+from mycroft.metrics import Stopwatch
+
+LOG.name = "MozillaRemoteTTS"
+
+
+class MozillaRemoteTTS(TTS):
+
+    def __init__(self, lang="en-us", config=None):
+        config = config or get_neon_tts_config().get("mozilla", {})
+        super(MozillaRemoteTTS, self).__init__(lang, config, MozillaTTSValidator(self),
+                                               audio_ext="mp3",
+                                               ssml_tags=["speak"])
+        self.base_url = config.get("api_url", "http://64.34.186.120:5002/api/tts")
```

Comment:
should not default to a random ip address, who knows who will inherit in the future?

Reply:
good catch; testing leftover that I should remove